### DISCUSSION TITLE
react-spectrum:TableView row any type

### DISCUSSION THREAD

**Discussion by birchrust** [adobe/react-spectrum#4737] [General]
How to avoid type errors
The code is an example in https://react-spectrum.adobe.com/react-spectrum/TableView.html

Error message:Element implicitly has type "any" because an expression of type "Key" cannot be used for index type "{ id: number; name: string; date: string; type: string; }".Could not find an index signature with an argument of type 'string' on type '{ id: number; name: string; date: string; type: string; }'.

**Comment by LFDanLu** [comment]
Mind sharing which example you are seeing this error on?
